docroot/modules/custom/mass_entity_usage/mass_entity_usage.services.yml

```diff
@@ -4,6 +4,3 @@ services:
     arguments: ['@entity_type.manager', '@config.factory']
     tags:
       - { name: event_subscriber }
-  mass_entity_usage.clean_usage_table:
```

Comment:
I see this service has been moved to an upstream patch. It's now available as `entity_usage.clean_usage_table`